### PR TITLE
Normalise warnings messages

### PR DESCRIPTION
**Pull request by johnnovak**
# Description

I've just done a warning message normalisations sweep for consistency. I like consistency, as you can tell 😎 

I left the `viewport_resolution` related warnings intentionally alone as I'm addressing those in my upcoming PR.

# Manual testing

DOSBox still compiles & starts up 😄 (no functional changes)

# Checklist

_Please tick the items as you have addressed them. Don't remove items; leave the ones that are not applicable unchecked._

I have:

- [x] followed the project's [contributing guidelines](https://github.com/dosbox-staging/dosbox-staging/blob/master/CONTRIBUTING.md) and [code of conduct](https://github.com/dosbox-staging/dosbox-staging/blob/master/CODE_OF_CONDUCT.md).
- [x] performed a self-review of my code.
- [ ] commented on the particularly hard-to-understand areas of my code.
- [x] split my work into well-defined, bisectable commits, and I [named my commits well](https://github.com/dosbox-staging/dosbox-staging/blob/main/CONTRIBUTING.md#commit-messages).
- [x] applied the appropriate labels (bug, enhancement, refactoring, documentation, etc.)
- [x] [checked](https://github.com/dosbox-staging/dosbox-staging/blob/main/scripts/compile_commits.sh) that all my commits can be built.
- [ ] confirmed that my code does not cause performance regressions (e.g., by running the Quake benchmark).
- [ ] added unit tests where applicable to prove the correctness of my code and to avoid future regressions.
- [ ] made corresponding changes to the documentation or the website according to the [documentation guidelines](https://github.com/dosbox-staging/dosbox-staging/blob/main/DOCUMENTATION.md).
- [ ] [locally verified](https://github.com/dosbox-staging/dosbox-staging/blob/main/DOCUMENTATION.md#previewing-documentation-changes-locally) my website or documentation changes.

### CODE REVIEW

File: src/capture/image/image_capturer.cpp [hunk 1]
```diff
@@ -55,14 +55,14 @@ void ImageCapturer::ConfigureGroupedMode(const std::string& prefs)
 
 	const auto formats = split_with_empties(prefs, ' ');
 	if (formats.size() == 0) {
-		LOG_WARNING("CAPTURE: 'default_image_capture_formats' not specified; "
-		            "defaulting to 'upscaled'");
+		LOG_WARNING("CAPTURE: 'default_image_capture_formats' not specified, "
+		            "using to 'upscaled'");
```
Comment (Torinde) [inline src/capture/image/image_capturer.cpp:59]: using to 'upscaled' -> using 'upscaled'?

File: src/capture/image/image_capturer.cpp [hunk 2]
```diff
@@ -55,14 +55,14 @@ void ImageCapturer::ConfigureGroupedMode(const std::string& prefs)
...
 		set_defaults();
 		return;
 	}
 	if (formats.size() > 3) {
 		LOG_WARNING("CAPTURE: Invalid 'default_image_capture_formats' setting: '%s'. "
-		            "Must not contain more than 3 formats; defaulting to 'upscaled'.",
+		            "Must not contain more than 3 formats, using to 'upscaled'.",
```
Comment (Torinde) [inline src/capture/image/image_capturer.cpp:65]: using to 'upscaled' -> using 'upscaled'?

File: src/capture/image/image_capturer.cpp [hunk 3]
```diff
@@ -76,10 +76,9 @@ void ImageCapturer::ConfigureGroupedMode(const std::string& prefs)
 		} else if (format == "rendered") {
 			grouped_mode.wants_rendered = true;
 		} else {
-			LOG_WARNING("CAPTURE: Invalid image capture format specified for "
-			            "'default_image_capture_formats': '%s'. "
+			LOG_WARNING("CAPTURE: Invalid 'default_image_capture_formats' setting: '%s'. "
 			            "Valid formats are 'raw', 'upscaled', and 'rendered'; "
-			            "defaulting to 'upscaled'.",
+			            "using to 'upscaled'.",
```
Comment (Torinde) [inline src/capture/image/image_capturer.cpp:81]: using to 'upscaled' -> using 'upscaled'?

File: src/gui/sdlmain.cpp [hunk 4]
```diff
@@ -2877,11 +2877,9 @@ static SDL_Point parse_window_resolution_from_conf(const std::string &pref)
 		return {w, h};
 	}
 
-	LOG_WARNING("DISPLAY: Requested window resolution '%s' is not valid, "
-	            "falling back to '%dx%d' instead",
-	            pref.c_str(),
-	            FallbackWindowSize.x,
-	            FallbackWindowSize.y);
+	LOG_WARNING("DISPLAY: Invalid 'windowresolution' setting: '%s', "
+	            "using 'default'",
```
Comment (Torinde) [inline src/gui/sdlmain.cpp:2881]: Why not logging anymore what the default/fallback was? %dx%d is removed along with FallbackWindowSize.x,            FallbackWindowSize.y

File: src/hardware/joystick.cpp [hunk 5]
```diff
@@ -534,8 +534,8 @@ static void configure_calibration(const Section_prop &settings)
 			return parsed_rates;
 		}
 		if (pref != "auto" && pref.length() != 0)
-			LOG_WARNING("JOYSTICK: Invalid %c_calibration parameters: %s."
-			            " Must be auto or number,number.",
+			LOG_WARNING("JOYSTICK: Invalid '%c_calibration' setting: '%s', "
+			            "using 'auto'",
```
Comment (Torinde) [inline src/hardware/joystick.cpp:538]: Why not notifying what the correct format is (number,number)? 
Further up a similar hint for a display setting is kept.

File: src/capture/image/image_capturer.cpp (hunk 2 above)
Comment (johnnovak) [reply src/capture/image/image_capturer.cpp:65]: fixed

File: src/capture/image/image_capturer.cpp (hunk 3 above)
Comment (johnnovak) [reply src/capture/image/image_capturer.cpp:81]: fixed

File: src/hardware/joystick.cpp (hunk 5 above)
Comment (johnnovak) [reply src/hardware/joystick.cpp:538]: I guess I can re-add.

File: src/gui/sdlmain.cpp (hunk 4 above)
Comment (johnnovak) [reply src/gui/sdlmain.cpp:2881]: Found it unnecessary.

**Review by kcgen — APPROVED** [review]

File: src/gui/sdlmain.cpp [hunk 6]
```diff
@@ -2225,8 +2225,8 @@ void GFX_SetMouseRawInput(const bool requested_raw_input)
 	if (SDL_SetHintWithPriority(SDL_HINT_MOUSE_RELATIVE_MODE_WARP,
 	                            requested_raw_input ? "0" : "1",
 	                            SDL_HINT_OVERRIDE) != SDL_TRUE)
-		LOG_WARNING("SDL: Mouse raw input %s failed",
-		            requested_raw_input ? "enable" : "disable");
+		LOG_WARNING("SDL: Failed %s raw mouse input",
+		            requested_raw_input ? "enabling" : "disabling");
```
Comment (weirddan455) [inline src/gui/sdlmain.cpp:2229]: "Failed to enable/disable raw mouse input" - I think reads better here.

**Review by weirddan455 — APPROVED** [review]
Just one minor wording suggestion

File: src/gui/sdlmain.cpp (hunk 6 above)
Comment (johnnovak) [reply src/gui/sdlmain.cpp:2229]: Changed.

File: src/hardware/joystick.cpp (hunk 5 above)
Comment (johnnovak) [reply src/hardware/joystick.cpp:538]: That message I removed was actually out of date. Simply going with my new version.

**Pull request merged**